site/content/docs/include/platform.ja.md

```diff
@@ -18,3 +18,18 @@ platform:
   osfamily: windows_server_2019_full
   architecture: x86_64
 ```
+```yaml
+platform:
+  osfamily: windows_server_2019_full
```

Comment:
Should this be `windows_server_2019_core`?

Reply:
Good catch.